### ISSUE TITLE
performance issue on right menu

### ISSUE THREAD

**Issue by hansimmmi** [aryaxt/iOS-Slide-Menu#123]
I have rather heavy right side menu, which has tableview and many small images. 
Once I see the right menu, the overall system performance drastically goes down. I don't see much changes in memory usage or CPU though. 

Is there anyway, if i can initiate the right menu every time I need to use and deallocate or deintiate as soon as closing it? Your advice will be highly appreciated!

**Comment by aryaxt** [comment]
I think that may make it slower, try this, and let me know if it makes a difference

when initializing the menu
```
menuViewController.view.layer.shouldRasterize = YES;
menuViewController.view.layer.rasterizationScale = [UIScreen mainScreen].scale;
```

**Comment by hansimmmi** [comment]
Thanks, but no luck. I found one workaround, it's a bit silly though.
Using the notification, when the right menu close, I bouce left menu, which somehow change right menu to left menu. My left menu has a few cells of string, so, the main view controller works fine now. It seems like only one view should exist.  So.. it turns out that the solution will be changing right view to left view without calling bounce menu, since left bounce is redundant upon closing right menu.

**Comment by hansimmmi** [comment]
It seems like it's one view or another. It would be nice if there is a way to change the view on the background without showing them.

**Comment by alexkotenko** [comment]
Yes I have this problem too. I add more small image on right menu  And the application Freese
